packages/locale/lang/my.ts

```diff
@@ -0,0 +1,184 @@
+export default {
+  name: 'my',
+  el: {
+    breadcrumb: {
+      label: 'ဘရတ်ဒ်ခရမ်',
+    },
+    colorpicker: {
+      confirm: 'အိုကေ',
+      clear: 'ရှင်းမယ်',
+      defaultLabel: 'အရောင်ရွေးချယ်ပါ',
+      description:
+        'လက်ရှိအရောင်မှာ {color} ဖြစ်ပါတယ်။ တခြားအရောင်ကိုရွေးချယ်လိုပါက enter ကိုနှိပ်ပါ။',
+      alphaLabel: 'alpha တန်ဖိုးကို ရွေးချယ်ပါ',
+    },
+    datepicker: {
+      now: 'ယခု',
+      today: 'ယနေ့',
+      cancel: 'ပယ်ဖျက်',
+      clear: 'ရှင်းမယ်',
+      confirm: 'အိုကေ',
+      dateTablePrompt:
+        'arrow keys နှင့် enter ကိုအသုံးပြုပြီး လတလရဲ့နေ့ရက်ကို ရွေးချယ်ပါ',
+      monthTablePrompt:
+        'arrow keys နှင့် enter ကိုအသုံးပြုပြီး လ ကိုရွေးချယ်ပါ',
+      yearTablePrompt:
+        'arrow keys နှင့် enter ကိုအသုံးပြုပြီး နှစ် ကိုရွေးချယ်ပါ',
+      selectedDate: 'ရွေးချယ်ထားတဲ့ ရက်စွဲ',
+      selectDate: 'ရက်စွဲကို ရွေးချယ်ပါ',
+      selectTime: 'အချိန်ကို ရွေးချယ်ပါ',
+      startDate: 'စတင်မည့်ရက်စွဲ',
+      startTime: 'စတင်မည့်အချိန်',
+      endDate: 'ကုန်ဆုံးမည့်ရက်စွဲ',
+      endTime: 'ကုန်ဆုံးမည့်အချိန်',
+      prevYear: 'ယခင်နှစ်',
+      nextYear: 'နောက်နှစ်',
+      prevMonth: 'ယခင်လ',
+      nextMonth: 'နောက်လ',
+      year: '',
+      month1: 'ဇန်နဝါရီ',
+      month2: 'ဖေဖော်ဝါရီ',
+      month3: 'မတ်',
+      month4: 'ဧပြီ',
+      month5: 'မေ',
+      month6: 'ဇွန်',
+      month7: 'ဇူလိုင်',
+      month8: 'သြဂုတ်',
+      month9: 'စက်တင်ဘာ',
+      month10: 'အောက်တိုဘာ',
+      month11: 'နိုဝင်ဘာ',
+      month12: 'ဒီဇင်ဘာ',
+      week: 'ရက်သတ္တပတ်',
+      weeks: {
+        sun: 'နွေ',
+        mon: 'လာ',
+        tue: 'ဂါ',
+        wed: 'ဟူး',
+        thu: 'ကြာ',
+        fri: 'သော',
+        sat: 'နေ',
+      },
+      weeksFull: {
+        sun: 'တနင်္ဂနွေ',
+        mon: 'တနင်္လာ',
+        tue: 'အင်္ဂါ',
+        wed: 'ဗုဒ္ဓဟူး',
+        thu: 'ကြာသပတေး',
+        fri: 'သောကြာ',
+        sat: 'စနေ',
+      },
+      months: {
+        jan: 'ဇန်',
+        feb: 'ဖေ',
+        mar: 'မတ်',
+        apr: 'ပြီ',
+        may: 'မေ',
+        jun: 'ဇွန်',
+        jul: 'လိုင်',
+        aug: 'ဩ',
+        sep: 'စက်',
+        oct: 'အောက်',
+        nov: 'နို',
+        dec: 'ဒီ',
+      },
+    },
+    inputNumber: {
+      decrease: 'အရေအတွက်လျှော့ချ',
+      increase: 'အရေအတွက်တိုး',
+    },
+    select: {
+      loading: 'ဝန်တင်နေသည်',
+      noMatch: 'ကိုက်ညီသောဒေတာမရှိပါ',
+      noData: 'ဒေတာမရှိပါ',
+      placeholder: 'ရွေးပါ',
+    },
+    mention: {
+      loading: 'ဝန်တင်နေသည်',
+    },
+    dropdown: {
+      toggleDropdown: 'Dropdown ဖွင့်/ပိတ်',
```

Comment:
Do the English words here need to be translated?

Reply:
@btea There isn't a direct translation for "Dropdown" in Burmese, but I’ve updated the translation to make it easier to understand.